### ISSUE TITLE
family = 'binomial' error

### ISSUE THREAD

**Issue by junpenglao** [bambinos/bambi#51]
Hi there,
I am getting a weird error after upgrading to 0.0.4.
I am modelling the accuracy of the behavioural response ([0, 1]) using:
```
acc_fitted = ACCmodel.fit('ACC ~ ViewCond*Race*Orit',random='1|Sbj',
                          family='binomial', samples=2000, njobs=2)
```
Which was fine before the upgrade but now I am getting the following error:

```
Traceback (most recent call last):

  File "<ipython-input-34-c6e18d364f43>", line 3, in <module>
    family='binomial', samples=2000, njobs=2)

  File "/usr/local/lib/python3.4/dist-packages/bambi/models.py", line 254, in fit
    self.build()

  File "/usr/local/lib/python3.4/dist-packages/bambi/models.py", line 206, in build
    self.y.name, str(self.data[self.y.name][event])))

  File "/usr/local/lib/python3.4/dist-packages/pandas/core/series.py", line 601, in __getitem__
    result = self.index.get_value(self, key)

  File "/usr/local/lib/python3.4/dist-packages/pandas/indexes/base.py", line 2139, in get_value
    tz=getattr(series.dtype, 'tz', None))

  File "pandas/index.pyx", line 105, in pandas.index.IndexEngine.get_value (pandas/index.c:3338)

  File "pandas/index.pyx", line 113, in pandas.index.IndexEngine.get_value (pandas/index.c:3041)

  File "pandas/index.pyx", line 161, in pandas.index.IndexEngine.get_loc (pandas/index.c:4024)

  File "pandas/src/hashtable_class_helper.pxi", line 404, in pandas.hashtable.Int64HashTable.get_item (pandas/hashtable.c:8141)

  File "pandas/src/hashtable_class_helper.pxi", line 410, in pandas.hashtable.Int64HashTable.get_item (pandas/hashtable.c:8085)

KeyError: 1
```

Any suggestion?

**Comment by jake-westfall** [comment]
I'm having some trouble reproducing this. Maybe you can post in this thread the result of the following:
`ACCmodel.data[['ACC', 'ViewCond', 'Race', 'Orit', 'Sbj']].head(50).to_json()`

Then I can look at that 50-row snippet of your dataset and see if I can reproduce the problem on that.

**Comment by junpenglao** [comment]
thanks!

```
'{"ACC":{"8":0,"9":1,"10":1,"11":1,"12":0,"13":1,"14":1,"15":1,"16":1,"17":1,"18":1,"19":1,"20":1,"21":0,"22":1,"23":1,"24":1,"25":1,"26":1,"27":1,"28":1,"29":1,"30":1,"31":1,"32":0,"33":1,"34":1,"35":1,"36":1,"37":0,"38":1,"39":1,"40":1,"41":1,"42":1,"43":1,"44":0,"45":0,"46":1,"47":0,"48":1,"49":1,"50":0,"51":1,"52":1,"53":1,"54":1,"55":1,"56":0,"57":1},"ViewCond":{"8":"NV","9":"NV","10":"NV","11":"NV","12":"NV","13":"NV","14":"NV","15":"NV","16":"NV","17":"NV","18":"NV","19":"NV","20":"NV","21":"NV","22":"NV","23":"NV","24":"NV","25":"NV","26":"NV","27":"NV","28":"NV","29":"NV","30":"NV","31":"NV","32":"NV","33":"NV","34":"NV","35":"NV","36":"NV","37":"NV","38":"NV","39":"NV","40":"NV","41":"NV","42":"NV","43":"NV","44":"NV","45":"NV","46":"NV","47":"NV","48":"NV","49":"NV","50":"NV","51":"NV","52":"NV","53":"NV","54":"NV","55":"NV","56":"NV","57":"NV"},"Race":{"8":"ea","9":"ea","10":"ea","11":"ea","12":"ea","13":"ea","14":"ea","15":"ea","16":"ea","17":"ea","18":"ea","19":"ea","20":"ea","21":"ea","22":"ea","23":"ea","24":"ea","25":"ea","26":"ea","27":"ea","28":"ea","29":"ea","30":"ea","31":"ea","32":"ea","33":"ea","34":"ea","35":"ea","36":"ea","37":"ea","38":"ea","39":"ea","40":"ea","41":"ea","42":"ea","43":"ea","44":"ea","45":"ea","46":"ea","47":"ea","48":"ea","49":"ea","50":"ea","51":"ea","52":"ea","53":"ea","54":"ea","55":"ea","56":"ea","57":"ea"},"Orit":{"8":"Up","9":"Inv","10":"Inv","11":"Up","12":"Up","13":"Up","14":"Inv","15":"Up","16":"Up","17":"Inv","18":"Inv","19":"Inv","20":"Up","21":"Up","22":"Inv","23":"Inv","24":"Inv","25":"Up","26":"Up","27":"Up","28":"Inv","29":"Up","30":"Up","31":"Inv","32":"Inv","33":"Up","34":"Up","35":"Up","36":"Inv","37":"Up","38":"Inv","39":"Up","40":"Up","41":"Inv","42":"Up","43":"Up","44":"Inv","45":"Inv","46":"Up","47":"Inv","48":"Inv","49":"Inv","50":"Inv","51":"Inv","52":"Up","53":"Inv","54":"Inv","55":"Up","56":"Up","57":"Up"},"Sbj":{"8":1,"9":1,"10":1,"11":1,"12":1,"13":1,"14":1,"15":1,"16":1,"17":1,"18":1,"19":1,"20":1,"21":1,"22":1,"23":1,"24":1,"25":1,"26":1,"27":1,"28":1,"29":1,"30":1,"31":1,"32":1,"33":1,"34":1,"35":1,"36":1,"37":1,"38":1,"39":1,"40":1,"41":1,"42":1,"43":1,"44":1,"45":1,"46":1,"47":1,"48":1,"49":1,"50":1,"51":1,"52":1,"53":1,"54":1,"55":1,"56":1,"57":1}}'
```

**Comment by jake-westfall** [comment]
Okay, I see the problem. Will push a fix shortly.

**Comment by jake-westfall** [comment]
Okay, this should be fixed by 99f796e46b328ada458bed04c957d97989dffb21. Reinstall from Github and let me know if that doesn't do the trick!

**Comment by junpenglao** [comment]
Yep no more error. Thanks!
